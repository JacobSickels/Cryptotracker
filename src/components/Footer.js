import React from 'react';

/*
    Footer is a React Component that is responsible for 
    rendering the footer contents on every page.
*/

const Footer = () => (
    <div className="footer">
        <div className="container">
            <p>If you are interested in how this project was made, check out the 
                <a href="./res/Capstone.pdf">Documentation</a></p>
        </div>
        <div className="footer-end">
            <div className="container">
                <p>This project was created by <a href="http://www.jacobsickels.com">Jacob Sickels</a></p>
            </div>
        </div>
    </div>
);

export default Footer;