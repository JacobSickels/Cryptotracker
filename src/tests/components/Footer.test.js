import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../components/Footer';

//This test makes sure the Footer component is rendered correctly
test('Should render footer correctly', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
});