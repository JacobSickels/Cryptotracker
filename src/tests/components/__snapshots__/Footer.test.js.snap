// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render footer correctly 1`] = `
<div
  className="footer"
>
  <div
    className="container"
  >
    <p>
      If you are interested in how this project was made, check out the
      <a
        href="./res/Capstone.pdf"
      >
        Documentation
      </a>
    </p>
  </div>
  <div
    className="footer-end"
  >
    <div
      className="container"
    >
      <p>
        This project was created by 
        <a
          href="http://www.jacobsickels.com"
        >
          Jacob Sickels
        </a>
      </p>
    </div>
  </div>
</div>
`;
